README.md

```diff
@@ -245,7 +245,7 @@ To listen for any new messages from _all_ conversations, use `conversations.stre
 > There is a chance this stream can miss messages if multiple new conversations are received in the time it takes to update the stream to include a new conversation.
 
 > **Warning**  
-> This stream will continue infinitely. To end the stream you can call `cancelStreamAllMessages()`.
+> This stream will continue infinitely. To end the stream you can call the method returned by `conversations.streamAllMessages()`.
 
 ```tsx
 for await (const message of await xmtp.conversations.streamAllMessages()) {
```

Comment:
This code example should be updated as well to show us save the cancel/unsubscribe function